res/compensation-benchmarks.yaml

```diff
@@ -0,0 +1,1477 @@
+openapi: "3.0.1"
```

Comment:
Will be this opeanapi specs pushed from vserver builds as other files do?

Reply:
Yes I believe there is a ticket open for that.